docusaurus/docs/migration-guides/update-from-grafana-versions/v8.3.x-8.4.x.md

```diff
@@ -2,7 +2,14 @@
 id: migrate-8.3.x-to-8.4.x
 title: 8.3.x to 8.4.x
 sidebar_position: 6
-description: "Follow these instructions to migrate Grafana v8.3.x plugins to the updated plugin system available in Grafana v8.4.x."
+description: How to migrate Grafana v8.3.x plugins to the updated plugin system available in Grafana v8.4.x.
```

Comment:
I'm not sure about "plugin system", but also it doesn't seem to be consistently applied in this section

Reply:
Fixed.